crates/swc/tests/fixture/issues-9xxx/9544/output/1.js

```diff
@@ -0,0 +1,4 @@
+require("core-js/modules/es.object.to-string.js");
+document.querySelectorAll('div').forEach(function(el) {
+    console.log(el);
+});
```

Comment:
@kdy1 to fix #9544, it should also load `web.dom-collections.for-each`, no?

Reply:
You are right. I'll reopen it.